pythonx/autosurround.py

```diff
@@ -338,6 +338,7 @@ def skip_matching_pair(open_pair, close_pair):
 
 
 def clean_current_pairs():
+    return
```

Comment:
why?

Reply:
it wasn't working, and I didn't see any difference after removing it, so...